middleware.js

```diff
@@ -35,9 +36,7 @@ function middleware (api, { baseIriFromRequest, loader, store, middleware = {} }
     }
 
     if (!api.term) {
-      const apiIri = new URL(iri)
-
-      apiIri.pathname = api.path
+      const apiIri = new URL(path.join(req.baseUrl, api.path), iri)
```

Comment:
Instead I propose to resolve the API Documentation URL only once when the API is initialised